src/lib.rs

```diff
@@ -448,7 +448,7 @@ fn mix<H: CollectionHandle>(
     });
     workload_scope.wait();
 
-    for (i, op) in (0..(ops / op_mix.len()))
+    for (i, op) in (0..((ops + op_mix.len() -1) / op_mix.len()))
```

Comment:
I think rustfmt will complain about the lack of a space after the `-` here

Reply:
My bad, I missed it.